docs/source/udf/buffer_types.py

```diff
@@ -0,0 +1,38 @@
+from libertem.udf import UDF
```

Comment:
Is it intentional that the file is indented by tabs? Usually, python code is indented by 4 spaces, but you can leave it as-is if there is a reason.